scripts/xtensa-build-zephyr.py

```diff
@@ -510,6 +514,10 @@ def build_platforms():
 
 		sign_cmd += ["-f", get_sof_version(abs_build_dir)]
 
+		if args.fw_naming == "AVS":
+			output_fwname="dsp_basefw.bin"
```

Comment:
sorry, we are loosing information here as all the FW files would have the same name. e.g. how could we tell platforms apart ?